app/views/jobs/getjob.html.erb

```diff
@@ -31,10 +31,10 @@
 </tr></table>
 
 <% if @cud.user.administrator?  then %>
-<li><kbd>notifyURL</kbd>: <kbd><%= @job[:rjob]["notifyURL"] %> </kbd>
+<li><kbd>notifyURL</kbd>: <%= @job[:rjob]["notifyURL"] %>
```

Comment:
@dlbucci I actually feel the same -- what do you suggest me to change this line into? I thought `Notify URL` might confuse people... What about `Callback URL`, or other options?